cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -370,7 +370,7 @@ func readDeviceConfiguration(pfDev string) string {
 		return defaultCapabilities
 	}
 
-	return devCfg.Section("GENERAL").Key("ServicesEnabled").String()
+	return strings.Join(strings.SplitN(strings.TrimSpace(devCfg.Section("GENERAL").Key("ServicesEnabled").String()), ";", 3), "-")
```

Comment:
The line is complicated enough that I think a helper function should be used both here, and on same operation done for on earlier line in this function => helper function could be internal to this function.

Reply:
I was slightly disappointed Go formatter could not do anything about it.